lib/pages/signup/pick-a-plan-page.js

```diff
@@ -39,10 +39,11 @@ export default class PickAPlanPage extends BaseContainer {
 	}
 	// Explicitly select the free button on jetpack without needing `host` above.
 	selectFreePlanJetpack() {
-		return driverHelper.clickWhenClickable(
-			this.driver,
-			By.css( '.jetpack-connect__plans-nav-buttons button' )
-		);
+		const diabledPersonalPlanButton = By.css( 'button[disabled].is-personal-plan' );
+		const freePlanButton = By.css( '.jetpack-connect__plans-nav-buttons button' );
+
+		return driverHelper.waitTillNotPresent( this.driver, diabledPersonalPlanButton )
```

Comment:
again, do these need to be chained?

Reply:
Same as above. If you think it not needed - I can revert it back

Reply:
no worries, happy to keep this